files/CCS_QUADBOX_POWEROFF

```diff
@@ -42,7 +42,7 @@ esac
 
 ## We could use plain ssh instead.
 echo "Shutting down remote hosts..."
-clush -w $quadbox\!$shost $poweroff -y
+clush -w "$quadbox\!$shost" "$poweroff" -y
```

Comment:
I don't think this works. If you want to make the code uglier to pacify shellcheck, you may need something like
"$quadbox"\!"$shost"
Since our hostnames don't contain spaces, I don't think this buys us anthing.

Reply:
I think we want the entire argument to `-w` inside of the same quotes since it is a string being passed to `clush` to be parsed. It does work the way you've suggested but it is passed as a single argument since there is no whitespace for the shell to split it on... so either way is equivalent.  E.g.:
```
[rke@pillan01 pillan (master)]$ clush -w "pillan01,pillan02\!pillan01" ls
pillan02\: cert-manager
pillan02\: ingress
pillan02\: metallb
pillan02\: multus
pillan02\: README.md
pillan02\: rke
pillan02\: rook-ceph
[rke@pillan01 pillan (master)]$ clush -w "pillan01,pillan02!pillan01" ls
-bash: !pillan01": event not found
[rke@pillan01 pillan (master)]$ clush -w "pillan01,pillan02"\!"pillan01" ls
pillan02: cert-manager
pillan02: ingress
pillan02: metallb
pillan02: multus
pillan02: README.md
pillan02: rke
pillan02: rook-ceph
```

Reply:
`clush -w "xoc2,xoc3,xoc4\!xoc4" hostname`
fails for me with:

> xoc4\: ssh: Could not resolve hostname xoc4\\: Name or service not known
> clush: xoc4\: exited with exit code 255

Reply:
Are you using bash?  The test above reports `4.2.46(2)-release`.

Reply:
Actually, something strange is going on with the escape in the first example too as the hostname is reported as `pillan02\'.

Reply:
`clush -w 'pillan01,pillan02!pillan01' ls` works as I would expect. I'll convert it to the form you suggested.  Not sure what is going on with character escaping inside double quotes.